modules/tinymce/src/core/main/ts/dom/DomSerializerImpl.ts

```diff
@@ -91,7 +92,6 @@ const DomSerializerImpl = (settings: DomSerializerSettings, editor?: Editor): Do
   const dom = editor && editor.dom ? editor.dom : DOMUtils.DOM;
   const schema = editor && editor.schema ? editor.schema : Schema(settings);
   settings.entity_encoding = settings.entity_encoding || 'named';
-  settings.remove_trailing_brs = 'remove_trailing_brs' in settings ? settings.remove_trailing_brs : true;
```

Comment:
Thinking about it, this may be a breaking change as well. This line defaults the `remove_trailing_brs` option to `true` within the Serializer. We should keep that behaviour but do it (along with `entity_encoding` defaulting) in a nicer way, such as:
```
  const defaultedSettings: DomSerializerSettings = {
    entity_encoding: 'named',
    remove_trailing_brs: true,
    ...settings
  };
```
and then using the `defaultedSettings` object rather than the `settings` object in the subsequent usages.